app/src/main/res/drawable/ic_stopwatch_24dp.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:viewportHeight="60.0"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_stopwatch_24dp` appears to be unused
 Unused resources make applications larger and slow down builds.